modules/miner/blockmanager_test.go

```diff
@@ -0,0 +1,105 @@
+package miner
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// reconstructBlock reconstructs a block given the miner and the header
+func reconstructBlock(m *Miner, header types.BlockHeader) (block *types.Block, err error) {
+	block, exists := m.blockMem[header]
+	if !exists {
+		err = errors.New("Header is either invalid or too old")
+		return
```

Comment:
for the most part, no. It's easier to make mistakes when you have an empty `return` line, for example you might not realize that you forgot to clear an error that was checked but not returned (this has happened a few times). If you explicitly return `nil`, it's clear that you intended to return nil.

This also removes guesswork from reviewing. A reviewer may look at such a mistake and think "he meant to do that". Even though the review has noticed the mistake, it is not fixed.

I think there are a few places where it makes sense. Mostly, really small functions where not using named returns would require adding multiple lines of additional code.